querybook/webapp/components/DataDocGitHub/GitHubVersions.tsx

```diff
@@ -88,64 +119,71 @@ export const GitHubVersions: React.FunctionComponent<IProps> = ({
         );
     }
 
+    const commitListDOM = (
+        <div className="commit-list">
+            {commitVersions.map((version) => (
+                <CommitCard
+                    key={version.sha}
+                    version={version}
+                    onRestore={handleRestore}
+                    onCompare={toggleCompare.bind(null, version)}
+                />
+            ))}
+        </div>
+    );
+
+    const loadMoreButtonDOM = hasMore ? (
+        <AsyncButton onClick={fetchMore} className="load-more-button">
+            Load More
+        </AsyncButton>
+    ) : null;
+
+    const comparePanelDOM = (
+        <div
+            className={`compare-slide-out-panel ${
+                isCompareOpen ? 'open' : ''
+            } ${isFullScreen ? 'full-screen' : ''}`}
+        >
+            {selectedCommit && (
+                <div className="GitHubVersionsComparePanel">
+                    <div className="panel-header">
+                        <IconButton
+                            icon={isFullScreen ? 'Minimize2' : 'Maximize2'}
+                            onClick={toggleFullScreen}
+                            size={16}
+                            tooltip={
+                                isFullScreen
+                                    ? 'Exit Full Screen'
+                                    : 'Enter Full Screen'
+                            }
+                            tooltipPos="left"
+                        />
+                        <IconButton
+                            icon="X"
+                            onClick={() => toggleCompare()}
+                            size={16}
+                            tooltip="Close Compare Panel"
+                            tooltipPos="left"
+                        />
+                    </div>
+                    <QueryComparison
+                        fromQuery={'Current DataDoc Old Text'}
```

Comment:
feel a bit confusing. when we say "Current", it is not the staging one in mysql (not committed), right?

Reply:
Oh I think current is the msql datadoc. I use current meaning the current datadoc the user is on in Querybook, and incoming as the incoming datadoc commit on GitHub. We can also maybe do:
```
<QueryComparison
    fromQuery={'Latest Committed DataDoc'}
    toQuery={'Selected Commit DataDoc'}
    fromQueryTitle="Latest Committed Version"
    toQueryTitle="Selected Commit Version"
/>
```

Reply:
the latest committed datadoc may not be the one in the mysql datadoc, when there are uncommitted changes.

and the name of "incoming datadoc" for the right side also doesn't feel right. those are actually history commits, not incoming changes. 

maybe we can swap the position, always put the older/history one on the left side and the current one on the right side. 
and for the left side title, maybe we  can include the commit title/time?